gcloud/search/index.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Define API Indexes."""
+
+
+class Index(object):
```

Comment:
The relevant index-only APIs are `list` (return info about a project's indexes) and `search` (return documents from a project matching a query):  I plan to implement `search` last.

@jgeewax seemed to indicate that the API might grow support for further index methods (there should at least be a `get` method, for instance) before GA.